src/SDPoptimize.jl

```diff
@@ -140,6 +140,59 @@ function solve_DP(model::SPModel,
     return V
 end
 
+"""
+Compute the value function at time t
```

Comment:
Reading the doc I could not understand what exactly the function is doing. I seems to be a bit late on this fact as it might be from a few PR ago.

A few comment : 
+ The general description is too short
+ the description of each parameter is unclear
+ I think that "randomness" is used for "noise"
+ I don't understand what is the "sampling size" 
+ I don't understand what are the samples